ui/dashboards/src/components/Dashboard/Dashboard.tsx

```diff
@@ -23,7 +23,7 @@ export type DashboardProps = BoxProps & {
    * is empty (i.e. has no panel groups). If not specified, the defaults will
    * be used.
    */
-  emptyDashboardProps: EmptyDashboardProps;
+  emptyDashboardProps?: EmptyDashboardProps;
```

Comment:
This prop is optional in other components, and it was an oversight that it wasn't optional here too.